applications/CompressiblePotentialFlowApplication/python_scripts/potential_flow_adjoint_solver.py

```diff
@@ -0,0 +1,90 @@
+from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+# importing the Kratos Library
+import KratosMultiphysics
+import KratosMultiphysics.CompressiblePotentialFlowApplication as KCPFApp
+from KratosMultiphysics.CompressiblePotentialFlowApplication.potential_flow_solver import PotentialFlowSolver
+import KratosMultiphysics.StructuralMechanicsApplication
```

Comment:
Do you still need to import the StructuralMechanicsApplication? I thought the purpose of the last changes was that we don't have dependencies with it.

Reply:
The only dependency on the structuralMechApp right now is the ReplaceElementAndCondition process. I think the idea is to move it to the core eventually but for now it will stay in the StructuralApp. Is this correct @armingeiser?

Reply:
@inigolcanalejo side comment, the dependency is only in python, therefore you will only need it if you use it
It is not the same dependency as you have on FluidDynamics

Reply:
Yes, the plan is to move this process to the core, but this will take some time for approval probably and until now you have to stick with the StructuralMechanicsApplication import.

Reply:
Ok, I was just wondering